setup.py

```diff
@@ -45,7 +45,14 @@
 #
 # Other demos do not have specific requirements
 EXTRAS_REQUIRES = {
-    "demos": ["numba", "jupyter", "seaborn", "rdflib", "mplleaflet==0.0.5"],
+    "demos": [
+        "numba",
+        "jupyter",
+        "seaborn",
+        "rdflib",
+        "mplleaflet==0.0.5",
+        "packaging",
```

Comment:
Does this work with conda? Installing via conda doesn't have an easy way to get these requirements (compared to `stellargraph[demos]` for pip), so does this package exist automatically there?

Reply:
interestingly, conda includes packaging by default but it can be forced out of an environment, so I've added it to our requirements (now for the library itself in pip and conda)